src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const empoyees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (string) => {
+  const arr = string.split('');
+
+  arr.shift();
+
+  const strNormalized = arr.filter((el) => el !== ',').join('');
+
+  return +strNormalized;
+};
+
+const sortList = (list) => {
+  const parentElement = list[0].closest('ul');
```

Comment:
You are assuming that all elements in the list have the same parent. This might not always be the case. It would be better to get the parent of each element individually inside the forEach loop.